Source/plugins/IController.h

```diff
@@ -215,7 +215,6 @@ namespace Controller {
                     RESUMED /* @text resumed */
                 };
 
-                bool AutoStart /* @brief Determines if the plugin is to be started automatically along with the framework */;
```

Comment:
@HaseenaSainul / @sebaszm agree it was not partof the old-code and it is not part of the PR we are resolving here but to align with the disucssion that @HaseenaSainul and I had, it would be good to also (netxt to the "startmode" have the "resumed" flag inhere as these two values together define the default startup behavior of a Service. So maybe good to add this falg (boolean) here as well ?

Reply:
done